bin/gql/resolver/Query/signup.js

```diff
@@ -41,5 +43,16 @@ export default async (parent, args, context, info) => {
     if (!user)
         throw new Error("Couldn't create user.");
 
+    new VerificationModel({
+        email: email
+    }).save();
+
+    await transporter.sendMail({
+        from: '"Vobe" <auth@vobe.io>',
+        to: email,
+        subject: "Please verify your account",
+        text: "Hey, you've just created an account on vobe.io. Please click on the link below to verify it. https://beta.vobe.io/verification/32093092",
```

Comment:
I don't think it's right that there is a fixed verification link in there.

Reply:
You're right, but the the verification feature isn't implemented in vobe-web so far

Reply:
Are you okay with the latest change?